gomint-server/src/main/java/io/gomint/server/world/generator/vanilla/VanillaGeneratorImpl.java

```diff
@@ -284,6 +288,17 @@ public VanillaGeneratorImpl(World world, GeneratorContext context) {
         }
     }
 
+    private int getRandomPort() {
+        return IntStream.range(50000, 60000).filter(port -> {
```

Comment:
Better: We could just use a while where we grab a random number up to 10000 times for testing.